src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Assets.targets

```diff
@@ -0,0 +1,147 @@
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+	created a backup copy.  Incorrect changes to this file will make it
+	impossible to load or build your projects from the command-line or the IDE.
+
+This file imports the version- and platform-specific targets for the project importing
+this file. This file also defines targets to produce an error if the specified targets
+file does not exist, but the project is built anyway (command-line or IDE build).
+
+Copyright (C) 2010-2011 Novell. All rights reserved.
+Copyright (C) 2011-2012 Xamarin. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+<UsingTask TaskName="Xamarin.Android.Tasks.AndroidComputeResPaths" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+<UsingTask TaskName="Xamarin.Android.Tasks.Aapt2LinkAssetPack" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+<UsingTask TaskName="Xamarin.Android.Tasks.GetAssetPacks" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+<UsingTask TaskName="Xamarin.Android.Tasks.RemoveUnknownFiles" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+<UsingTask TaskName="Xamarin.Android.Tasks.CreateDynamicFeatureManifest" AssemblyFile="Xamarin.Android.Build.Tasks.dll" />
+
+<!-- Assets build properties -->
+<PropertyGroup>
+		<MonoAndroidAssetsDirIntermediate>$(IntermediateOutputPath)assets\</MonoAndroidAssetsDirIntermediate>
+		<MonoAndroidAssetPacksDirIntermediate>$(IntermediateOutputPath)assetpacks</MonoAndroidAssetPacksDirIntermediate>
+		<MonoAndroidAssetsPrefix Condition="'$(MonoAndroidAssetsPrefix)' == ''">Assets</MonoAndroidAssetsPrefix>
+		<AndroidIncludeAssetPacksInPackage Condition=" '$(AndroidIncludeAssetPacksInPackage)' == '' ">true</AndroidIncludeAssetPacksInPackage>
+</PropertyGroup>
+
+<PropertyGroup>
+	<BeforeBuildAndroidAssetPacks>
+		UpdateAndroidAssets
+		;_CalculateAssetsWithAssetPackMetaData
+		;_CalculateAssetPacks
+		;$(BeforeBuildAndroidAssetPacks)
+		;_CreateAssetPackManifests
+	</BeforeBuildAndroidAssetPacks>
+</PropertyGroup>
+
+<!-- Assets Build -->
+
+<Target Name="UpdateAndroidAssets"
+	DependsOnTargets="$(CoreResolveReferencesDependsOn);_ComputeAndroidAssetsPaths;_CalculateAssetPacks;_GenerateAndroidAssetsDir" />
+
+<Target Name="_ComputeAndroidAssetsPaths">
+	<AndroidComputeResPaths
+		ResourceFiles="@(AndroidAsset)"
+		IntermediateDir="$(MonoAndroidAssetsDirIntermediate)"
+		AssetPackIntermediateDir="$(MonoAndroidAssetPacksDirIntermediate)"
+		Prefixes="$(MonoAndroidAssetsPrefix)"
+		ProjectDir="$(ProjectDir)"
+	>
+		<Output ItemName="_AndroidAssetsDest" TaskParameter="IntermediateFiles" />
+		<Output ItemName="_AndroidResolvedAssets" TaskParameter="ResolvedResourceFiles" />
+	</AndroidComputeResPaths>
+</Target>
+
+<Target Name="_GenerateAndroidAssetsDir"
+		Inputs="@(_AndroidMSBuildAllProjects);@(_AndroidResolvedAssets)"
```

Comment:
*Here* you indented the attributes an extra tab stop, which is good!

Pity it's tabs and not spaces… ;-)